library/Rules/PostalCode.php

```diff
@@ -113,7 +113,7 @@ class PostalCode extends Regex
         "NF" => "/^(\d{4})$/",
         "NG" => "/^(\d{6})$/",
         "NI" => "/^(\d{7})$/",
-        "NL" => "/^(\d{4}[A-Z]{2})$/",
+        'NL' => "/^(\A[1-9]\d{3} ?[a-zA-Z]{2}\z)$/",
```

Comment:
It's PostNL, they don't provide any information on this. Never mind the change then.